src/hooks/api/places.ts

```diff
@@ -221,14 +288,22 @@ const addPlace = async ({
   address,
   mainLanguage,
   name,
-  type,
+  terms,
+  workflowStatus,
 }: AddPlaceArguments) =>
   fetchFromApi({
-    path: `/place`,
+    path: `/places`,
     options: {
       method: 'POST',
       headers,
-      body: JSON.stringify({ calendar, address, mainLanguage, name, type }),
+      body: JSON.stringify({
+        calendar,
```

Comment:
I think we'll also have to refactor this like on event.

Reply:
Oh yes, `terms` instead `type` you mean?

Reply:
I was talking about not nesting it in calendar:
Also I noticed we were not passing openingHours anywhere

https://github.com/cultuurnet/udb3-frontend/pull/484/commits/07f58cc7c08cb9d8792d296d5772e2adb3f64e5f